test/include/test_runner/test_group.h

```diff
@@ -10,15 +10,21 @@ struct TestStatement {
     std::string query;
     uint64_t numThreads = 4;
     std::string encodedJoin;
-    uint64_t expectedNumTuples = 0;
     bool expectedError = false;
+    std::string errorMessage;
     bool expectedOk = false;
+    uint64_t expectedNumTuples = 0;
     std::vector<std::string> expectedTuples;
-    std::string errorMessage;
     bool enumerate = false;
     bool checkOutputOrder = false;
-    bool isBeginWriteTransaction = false;
     std::string expectedTuplesCSVFile;
+    enum class TransactionType {
```

Comment:
I don't think transactionType is an accurate name for this. We only have read/write transactioin.
Maybe call it transactionCmdType? 
By the way, when do we need the NONE transaction type?

Reply:
Thanks. I'll change to `transactionCmdType`.
I have to set a default value (NONE), otherwise enum class will set to the first value (WRITE).

Reply:
We have a transacitonType called `auto commit`.
U can rename NONE->`AUTO COMMIT`